tests/test_browser.py

```diff
@@ -5083,6 +5097,14 @@ def test_wasm2js_fallback_on_wasm_compilation_failure(self):
   def test_system(self):
     self.btest_exit(test_file('system.c'))
 
+  # Tests building with -sSHARED_MEMORY
+  @also_with_minimal_runtime
+  def test_shared_memory(self):
+    self.btest(path_from_root('tests', 'wasm_worker', 'shared_memory.c'), expected='0', args=[])
+    self.btest(path_from_root('tests', 'wasm_worker', 'shared_memory.c'), expected='1', args=['-sSHARED_MEMORY'])
+#    self.btest(path_from_root('tests', 'wasm_worker', 'shared_memory.c'), expected='1', args=['-sWASM_WORKERS'])
+    self.btest(path_from_root('tests', 'wasm_worker', 'shared_memory.c'), expected='1', args=['-pthread'])
```

Comment:
`test_file('wasm_worker/shared_memory.c')`

Can this just be in test_other.py?   I don't see why this should be a test.

Reply:
Yes it could..